2333/2/libs/analyzer_printer.rb

```diff
@@ -0,0 +1,43 @@
+require_relative './top_bad_words_analyzer.rb'
+require_relative './top_words_analyzer.rb'
+
+module AnalyzerPrinter
```

Comment:
I think it's better to split this module to a couple of classes. Prefer using objects and classes not modules.